project.clj

```diff
@@ -32,15 +33,30 @@
   :profiles {
              ;; Clojure versions matrix
              :provided {:dependencies [[org.clojure/clojure "1.10.0"]]}
-             :1.8 {:dependencies [[org.clojure/clojure "1.8.0"]]}
-             :1.9 {:dependencies [[org.clojure/clojure "1.9.0"]]}
-             :1.10 {:dependencies [[org.clojure/clojure "1.10.0"]]}
+             :1.8 {:dependencies [[org.clojure/clojure "1.8.0"]
+                                  [org.clojure/clojurescript "1.10.520" :scope "provided"]]}
```

Comment:
@cichli @bbatsov
As you know the Cljs version previously tested was matching the Clojure minor version (`1.8.0` -> `1.8.51`). 

Unfortunately exactly `1.8.51` has a problem, it is so so old that it is not even reading `clojure.core.async` from the classpath when compiling - probably a bug. I tested a couple of `1.9`s on Maven and it seems like the fix was around the `1.9.2XX` series.

I had to make a choice here. In my daily ClojureScript, the most recent version is the one that I use. I personally have never ever even thought of using something ancient like `1.8.51`.

Does it then make sense what I did there - aka support only the latest?

All the tests are green this way but I am going to wait for feedback here.

Reply:
I wonder if anyone uses ClojureScript other than the most recent version. Not a big deal.